lessons/html-and-css/intro-to-html-and-css-part-1/html-exercises.md

```diff
@@ -0,0 +1,63 @@
+<!-- @acxbank html-tags-h -->
+### Q1. html-tags-h
+
+Which six tags are used to create headings and sub-heading of varying sizes in an HTML document?
+Write the tags in order from largest font to smallest font.
+<!-- end @acxbank -->
+<!-- @acxbank html-tags-p -->
+### Q2. html-tags-p
+
+Which tags would you use to cerate three separate paragraphs in your HTML document?
+<!-- end @acxbank -->
+<!-- @acxbank html-tags-br -->
+### Q3. html-tags-br
+
+Which HTML tag creates a line break in your text? How is it different from other tags such as `<p>`?
+<!-- end @acxbank -->
+<!-- @acxbank html-tags-button -->
+### Q4. html-tags-button
+
+Write the html code to create a button that has no default behvaior. Give the button the name "click-me" and the label "click here!"
+<!-- end @acxbank -->
+<!-- @acxbank html-tags-button-2 -->
+### Q5. html-tags-button-2
+
+Think of how you could create a button that, when clicked on, links to www.google.com. Give the button the name "google-btn" and display the string *google* on the button.
+
+Write the html code to two ways to accomplish this.
```

Comment:
`Write the html code to build this in two different ways` is probably what you mean?